html/controls/plt/plt-input.css

```diff
@@ -13,7 +13,12 @@ table.PLT, .PLT table, .PLT tr, .PLT td, #PenaltyEditor table, #PenaltyEditor tr
 	width: 99%;
 	top: 0%;
 }
-.PLT.Team thead td { font-size: 1em; font-weight: bold; background-color: #222; color: #FFF; }
+/* Colors are the same as the operator controls, so per-team colours look the same on both. */
+.PLT.Team thead td { font-size: 200%; font-weight: bold; background-color: #cccccc; color: #333333; }
```

Comment:
This font-size breaks the screen on typical tablet resolutions.

Reply:
Bad copy on my part, I prefer the smaller size. Fixed.